lib/ecto/postgres/pg_rational_migrations.ex

```diff
@@ -0,0 +1,132 @@
+use Vtc.Ecto.Postgres.Utils
+
+defpgmodule Vtc.Ecto.Postgres.PgRational.Migrations do
+  @moduledoc """
+  Migrations for adding rational types, functions and constraints to a
+  Postgres database.
+  """
+  alias Ecto.Migration
+
+  require Ecto.Migration
+
+  @doc section: :migrations_full
+  @doc """
+  Adds raw SQL queries to a migration for creating the database types, associated
+  functions, casts, operators, and operator families.
+
+  Includes all migraitons under the
+  [PgTypes](Vtc.Ecto.Postgres.PgRational.Migrations.html#pgtypes) and
+  [PgFunctions](Vtc.Ecto.Postgres.PgRational.Migrations.html#pgfunctions)
+  headings.
```

Comment:
```suggestion
  Includes all migrations under the
  [PgTypes](Vtc.Ecto.Postgres.PgRational.Migrations.html#pgtypes) and
  [PgFunctions](Vtc.Ecto.Postgres.PgRational.Migrations.html#pgfunctions)
  headings.
```

...though i think this line can simplify now that pgfunctions are not in this PR

Reply:
In 1668a5bd05ea95acdf1013a3370b07ccc2af55f7